webapp/components/admin_console/storage_settings.jsx

```diff
@@ -258,6 +262,43 @@ export default class StorageSettings extends AdminSettings {
                     onChange={this.handleChange}
                     disabled={!this.state.enableFileAttachments}
                 />
+                <BooleanSetting
+                    id='EnableClientSideEncryption'
+                    label={
+                        <FormattedMessage
+                            id='admin.file.enableClientSideEncryptionTitle'
+                            defaultMessage='Encrypt Files at the Mattermost server'
```

Comment:
I think it's just disabled but still visible when local file storage is selected. I think it's fine to still display it when local file storage is enabled, but it should be clearly grouped with the other S3 settings